app/views/provider/admin/dashboards/widgets/_chart.html.erb

```diff
@@ -1,7 +0,0 @@
-<div class="Dashboard-chart" data-chart>
-  <% if widget.loaded? %>
```

Comment:
After taking a deeper look at how widgets currently work, I think `loaded` just means whether the data is available or not. I don't see any indication of an async process, so rendering a spinner like in https://github.com/3scale/porta/blob/7f8fcd9276bd3708a10bbda07e9f14ffd4301985/app/views/provider/admin/dashboards/widgets/_potential_upgrades.html.slim#L26 does little sense to me. To me it would be more appropriate to show a "data not available" sign instead.